roles/edpm_ovn/meta/argument_specs.yml

```diff
@@ -249,8 +241,8 @@ argument_specs:
       edpm_ovn_metadata_common_volumes:
         default:
         - /lib/modules:/lib/modules:ro
-        - /run/openvswitch:/run/openvswitch:z
-        - '{{ edpm_ovn_neutron_config_dir }}/etc/neutron:/etc/neutron:z'
+        - /run/openvswitch:/usr/local/var/run/openvswitch:z
```

Comment:
couldn't get how change to /usr/local/var/run/openvswitch is related here with this change.

Reply:
I noticed that ovs library cannot connect to ovs, see where it connects in logs

```
2023-09-12 14:07:17.947 2 INFO ovsdbapp.backend.ovs_idl.vlog [-] unix:/usr/local/var/run/openvswitch/db.sock: connecting...                                                             
2023-09-12 14:07:17.948 2 INFO ovsdbapp.backend.ovs_idl.vlog [-] unix:/usr/local/var/run/openvswitch/db.sock: connected                                                                 
```

But yes, it's not directly related to the PR goal (which is primarily - load config-dir).

Reply:
That said, I just noticed this:

```
2023-09-12 19:13:59.883 2 DEBUG neutron.agent.ovn.metadata_agent [-] ovn.vhost_sock_dir             = /var/run/openvswitch log_opt_values /usr/lib/python3.9/site-packages/oslo_config/cfg.py:2609
```

So I wonder if we should set up ovsdbapp - or the vhost-sock-dir setting? - to sync them?

Reply:
AFAIU this comes from neutron/conf/agent/ovn/metadata/config.py

```
    cfg.StrOpt('ovsdb_connection',
               default='unix:/usr/local/var/run/openvswitch/db.sock',
               help=_('The connection string for the native OVSDB backend. '
                      'Use tcp:IP:PORT for TCP connections. '
                      'Use unix:FILE for unix domain socket connections.')),

```

Reply:
Nevermind, I think the errors I saw was due to the final config file not loaded, which lists `ovsdb_connection = tcp:127.0.0.1:6640`. So this change is not needed. I will remove it.